### ISSUE TITLE
Tensorboard doesn't show WER, loss on training step

### ISSUE THREAD

**Issue by ambigus9** [mozilla/DeepSpeech#1947]
- **TensorFlow version**: 1.12.0
- **Python version**: 3.6.3

I would like to see loss over each ephocs in training process like is showing in this [pull](https://github.com/mozilla/DeepSpeech/pull/1391) 

Currently I'm trying as follows

`tensorboard --logdir=/home/miguel/DeepSpeech/ --summary_dir=/home/miguel/DeepSpeech/`

but i'm getting this error: 

`tensorboard: error: unrecognized arguments: --summary_dir=/home/miguel/DeepSpeech`

Any idea to solve?

**Comment by reuben** [comment]
The summary_dir parameter is passed to DeepSpeech.py, not Tensorboard. Tensorboard takes just the --logdir parameter (which should be the same value as you passed to DeepSpeech.py)

**Comment by ambigus9** [comment]
Thanks! It works! Here is the code i'm using:

```
python -u DeepSpeech.py \
...
--summary_dir /home/miguel/DeepSpeech \

tensorboard --logdir=/home/miguel/DeepSpeech/
```
